src/api/userManagementApi.ts

```
@@ -216,113 +221,185 @@ export const createUser = async (userData: User): Promise<User> => {
-          profession: userData.profession || '',
-          parentNo: userData.parentNo || '',
-          studentAdmissionNo: userData.studentAdmissionNo || '',
-          relation: 'Guardian',
-        }
+      // Create parentData object with all required fields
+      const parentData: Record<string, any> = {
+        relation: 'Guardian',
+        studentAdmissionNo: userData.studentAdmissionNo || '',
+        profession: userData.profession || '',
+        contact: userData.parentContact || '' // Change parentContact to contact
       };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)